### DISCUSSION TITLE
How to store ROI segmentation information

### DISCUSSION THREAD

**Discussion by bendichter** [dandi/helpdesk#41] [Q&A]
I met with Bridget Wicinski (@wicinb01) about uploading data to DANDI. Their group is producing manual segmentation annotations for 3D images in [dandiset 26](https://dandiarchive.org/dandiset/000026/draft). Some examples of their annotations are [here](https://drive.google.com/file/d/1ph-PQC1kE29o2jQzj1gRaKWVVuBlOUrN/view?usp=sharing). I gather that this is a proprietary data format from StereoInvestigator by MBF Bioscience. Since the data is anatomical and should maybe use BIDS like the source data, this is a bit outside my wheelhouse. What would be the best way for them to format their data? Should they be uploaded into dandiset 26 or in a new dandiset?

**Comment by satra** [comment]
@bendichter - some of this is already in dandiset 26 under derivative datasets. (e.g., https://gui.dandiarchive.org/#/dandiset/000026/draft/files?location=derivatives%2FMBF_stereology%2F). what would be nice is to organize (i.e. rename, subject folder) like a proper bids dataset with appropriate json files highlighting which source files the annotations came from. for this dataset, it makes most sense to keep it with the raw data, and these are very lightweight files relative to the actual images.

**Reply by bendichter** [reply]
@wicinb01 (Bridget),

What is the data you wanted to upload to DANDI? Is it the .dat files or .xls files? As @satra points out there is precedent for just uploading xls files, so your data may not require standardization as I had originally thought, and can be uploaded directly to DANDI as-is.

Either way, whoever is uploading this data needs to be listed as an owner of dandiset 26. The current owners are JiaruiYang, Giacomo Mazzamuto, and lmorgan6.
